examples/ui/size_constraints.rs

```diff
@@ -293,6 +300,15 @@ fn spawn_button(
         });
 }
 
+fn button_activated(
+    mut button_activated_event: EventWriter<ButtonActivatedEvent>,
+    mut click_events: EventReader<Click>,
+) {
+    for event in &mut click_events.read() {
+        button_activated_event.send(ButtonActivatedEvent(event.0));
```

Comment:
Sorry, I meant to remove `ButtonActivatedEvent` and use `Click` instead.

Reply:
Oh sorry yeah that makes more sense